cl/assets/templates/includes/new_pagination.html

```diff
@@ -0,0 +1,83 @@
+{% load humanize %}
+{% load extras %}
+<div class="col-xs-12" >
+  {% if page_obj.has_other_pages %}
+    <div class="well v-offset-above-3 hidden-print">
+      <div class="row">
+	      <div class="col-xs-2">
+          {% if page_obj.has_previous %}
+            {% if not hide_first %}
+              <a href="?{{ get_string }}page=1"
+                 rel="first" class="btn btn-default" >
+                <i class="fa fa-caret-left no-underline" ></i>
+                <i class="fa fa-caret-left no-underline" ></i>
+			          <span class="hidden-xs hidden-md">First</span>
+              </a>
+            {% endif %}
+            <a href="?{{ get_string }}page={{ page_obj.previous_page_number }}"
+               rel="prev"
+               class="btn btn-default" >
+              <i class="fa fa-caret-left no-underline" ></i>
+			        <span class="hidden-xs hidden-md">Prev.</span>
+            </a>
+          {% else %}
+            {% if not hide_first %}
+              <span class="btn btn-default" >
+                <i class="fa fa-caret-left no-underline" ></i>
+                <i class="fa fa-caret-left no-underline" ></i>
+			          <span class="hidden-xs hidden-md">First</span>
+              </span>
+            {% endif %}
+            <span class="btn btn-default" >
+              <i class="fa fa-caret-left no-underline" ></i>
+			        <span class="hidden-xs hidden-md">Prev.</span>
+            </span>
+          {% endif %}
+        </div>
+        <div class="col-xs-8 text-center">
+          {% if not hide_numbers %}
+            {% for page in page_obj|page_prev_range %}
+              <a class="btn btn-default" href="?{{ get_string }}page={{ page }}">{{page}}</a>
+            {% endfor %}
+            <span class="btn btn-default disabled"> {{ page_obj.number }}</span>
+            {% for page in page_obj|page_next_range %}
+              <a class="btn btn-default" href="?{{ get_string }}page={{ page }}">{{page}}</a>
+            {% endfor %}
+          {% else %}
+            Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}
```

Comment:
The old one had this:

```suggestion
            <span class="hidden-xs">Page </span> {{ page_obj.number|intcomma }} of {{ page_obj.paginator.num_pages|intcomma }}
```

Seems like the intcomma and hiding the word "Page" are still good ideas?